### PR TITLE
Add Travis config

### PR DESCRIPTION
**Pull request by natebosch**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+language: dart
+sudo: false
```
Comment (kevmoo) [inline .travis.yml]: remove this line

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+dart:
+  - stable
+  - dev
+cache:
```
Comment (kevmoo) [inline .travis.yml]: move to the end

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  directories:
+    - $HOME/.pub-cache
+
+dist: trusty
```
Comment (kevmoo) [inline .travis.yml]: needed?

Comment (natebosch) [reply .travis.yml]: not sure, cargo culted :)

Will try removing it

File: .travis.yml (hunk 1 above)
Comment (natebosch) [reply .travis.yml]: Done

File: .travis.yml (hunk 2 above)
Comment (natebosch) [reply .travis.yml]: Done

**Pull request merged**